packages/fonts/README.md

```diff
@@ -0,0 +1,27 @@
+# HeyCar-UIKit - Fonts
+
+The Fonts is a web fonts kit and is used host heycar fonts
+It hosts web fonts for heycar locally. It adds objecttiv family fonts to all available sizes and are in [ttf format](https://en.wikipedia.org/wiki/TrueType)
+It hosts all fonts in static folder we support thesse fonts with @font-face
+
+## Installation
+
+To install and save in your package.json dependencies, run the command below using npm:
+
+```bash
+npm install @heycar-uikit/fonts
+```
+
+## Usage
+
+```tsx
+import '@heycar-uikit/fonts';
+
+p {
+     font-family: 'Objektiv';
+}
+
+h1 {
+     font-family: 'Objektiv';
+}
+```
```

Comment:
Is it a strange example, is not it?
Do you import the empty javascript file in the javascript and after that you write the CSS into the javascript file?